README.markdown

```diff
@@ -432,6 +429,29 @@ Default: true
 Adds the smuxpeer directive to the snmpd.conf file to allow net-snmp to talk with Dell's OpenManage.
 Default: false
 
+##### `master`
+Allow setting the *master* option, typically to enable AgentX registrations.
+Default: agentx
```

Comment:
Should be "Default: false" based on the actual value in params.pp. And should probably be worded differently.